src/app/RecordingList/ActiveRecordingsList.tsx

```diff
@@ -122,6 +128,34 @@ export const ActiveRecordingsList: React.FunctionComponent<ActiveRecordingsListP
     );
   }, []);
 
+  React.useEffect(() => {
+    const created_sub = context.notificationChannel.messages(RECORDING_CREATED)
+      .subscribe(v => {
+        const recordingName = Object.values(v.message)[0];
+        const targetId = Object.values(v.message)[1];
+        notifications.info('WebSocket Client Activity', `Recording ${recordingName} created in target: ${targetId}`);
+        refreshRecordingList();
+      });
+      const archived_sub = context.notificationChannel.messages(RECORDING_ARCHIVED)
+      .subscribe(v => {
+         const recordingName = Object.values(v.message)[0];
+         notifications.info('WebSocket Client Activity', `Recording ${recordingName} archived`);
```

Comment:
And on the topic of the notification message itself, the title `WebSocket Client Activity` was meant to indicate that a WebSocket client had (dis)connected, so that a user would know if another user had connected another remote session, or to give an individual user some notification if their connection to the server died - it wasn't intended as a catch-all title for all server-side notifications.

I think I would prefer to see a title for this event like `Recording Archived` and the message body say `${recordingName} was archived`. What do you think?

Reply:
Makes sense to me, I've changed this now